protozfits/__init__.py

```diff
@@ -1,5 +1,4 @@
 from pkg_resources import resource_string
-from .digicam import File as DigicamFile
 from .simple import File as SimpleFile
```

Comment:
from .simple import File

Reply:
actually one should rename the `simple.py` file to something more explicit

Reply:
Thank you for this review. For sure all this will come, but at the moment I am hunting a very interesting Bug, that revealed itself as I removed the file digicam.py.

It is a segmentation violation.

This normally does not occur in python anymore, usually it comes from underlying C code. However this segfault only showed up, after I removed a Python file from the project.

So a missing Python file triggered somehow a bug/feature in the underlying C code, which results in a segfault...

And now it comes.... this happens only on OSX, so for me it is *very hard* to track down the reason for this behavior and find its source.

Reply:
sorry I can't help you with this

Reply:
c.f. #39